Plugins/NativeXr/Source/NativeXr.cpp

```diff
@@ -1186,15 +1189,34 @@ namespace Babylon
                 env.Global().Set(JS_CLASS_NAME, func);
             }
 
-            static Napi::Promise CreateAsync(const Napi::CallbackInfo& info)
+            static Napi::Promise CreateAsync(const Napi::CallbackInfo& info, const JsRuntimeScheduler& runtimeScheduler)
```

Comment:
I think rather than passing in a scheduler to this function (seems weird) I would just have `XRSession` own it's own `JsRuntimeScheduler`.

Reply:
Done